src/Widgets/HeaderBar.vala

```diff
@@ -98,7 +81,7 @@ public class Camera.Widgets.HeaderBar : Gtk.HeaderBar {
         var take_button_style_provider = new Gtk.CssProvider ();
 
         try {
-            take_button_style_provider.load_from_data (TAKE_BUTTON_STYLESHEET, -1);
+            take_button_style_provider.load_from_resource ("/io/elementary/camera/application.css");
         } catch (Error e) {
             warning ("Styling take button failed: %s", e.message);
         }
```

Comment:
Since this is `from_resource` instead of `from_data`, I think this is now an unreachable catch. So it doesn't need the try/catch statement anymore :)

Reply:
yeah I did not notice that. thanks